src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String password;
```

Comment:
The 'password' field should have an @Column annotation to specify the column details in the database. This is important for clarity and to avoid issues with column naming.